funnel/forms/proposal.py

```diff
@@ -60,6 +60,15 @@ class ProposalForm(forms.Form):
     location = forms.StringField(__("Your location"), validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)],
         description=__("Your location, to help plan for your travel if required"))
 
+    def __init__(self, *args, **kwargs):
+        super(ProposalForm, self).__init__(*args, **kwargs)
+        space = kwargs.get('parent')
+        self.objective.label.text = space.labels['proposal']['partA']['title']
+        self.objective.description = space.labels['proposal']['partA']['hint']
+        self.description.label.text = space.labels['proposal']['partB']['title']
+        self.description.description = space.labels['proposal']['partB']['hint']
```

Comment:
I'd put all four in defensive `if` statements so that we set the label and description only if they are present in `space.labels`. It's a dict, so it's entirely possible to have a scenario where the values are missing.